msteams-platform/concepts/deploy-and-publish/overview.md

```diff
@@ -1,56 +1,65 @@
+## Publish to your organization
 
-If you want to distribute your app more broadly,Teams provides an in-app gallery for users to find or discover high-quality Teams apps. To have your solution available in the gallery, you must either [publish to your organization's app catalog](#publish-to-your-organizations-app-catalog) or [publish to AppSource](./appsource/publish.md).
+The steps to publish your apps can vary depending on the organization to which you are publishing them.
 
-### Publish to your organization's app catalog
+### Enterprise organization
+The publishing to your organization's app catalog can only be managed by Teams users with Microsoft Office 365 tenant admin privileges.
 
-Your organization's app catalog contains apps unique to your organization and is completely under your organization's control. You can find more information in the article [*Publish apps to your organization's app catalog*](/microsoftteams/tenant-apps-catalog-teams). This feature can only be managed by Teams users with Microsoft Office 365 tenant admin privileges.
+**To publish to your organization's app catalog**
 
-### Publish to AppSource
+Your organization's app catalog contains apps that are unique to your organization and is completely under your organization's control. For more information on publishing the app to your organization's app catalog, see [manage your apps in the Microsoft Teams admin center](/microsoftteams/tenant-apps-catalog-teams). 
 
-AppSource (formerly known as Office Store) provides a convenient location for you to distribute your Microsoft Teams app, as well as other Office 365 extensibility types such as Office add-ins and SharePoint add-ins. Follow our guidelines to [submit your app to AppSource](./appsource/publish.md).
+### Government Community Cloud (GCC) organization
+If your organization is a member of the GCC then it is GCC tenant administrator who decides if you can upload or publish an app.
 
-## Government Community Cloud (GCC) organizations
-
-### Upload your custom app directly to Teams
+> [!NOTE]
+> Microsoft does not own or control your custom applications, therefore, you must ensure that all endpoints are compliant with your organization's requirements. In addition, if the app solution includes a bot or message extension, you must complete the [Bot Framework registration](https://dev.botframework.com/).
 
- As a GCC tenant administrator, you'll decide whether to upload a custom app to your tenant environment and whether to  publish it to your tenant app catalog. Microsoft doesn't own or control your custom applications, therefore, you must ensure that all endpoints are compliant with your organization's requirements. Additionally, if the app solution includes a bot or message extension, you'll need to complete the [Bot Framework](https://dev.botframework.com/) registration as follows:
+**To configure a Government Community Cloud (GCC) distribution**
```

Comment:
This is only for apps with bots and messaging extensions, so we should probably clarify.